core/src/UI/CoreUIParent.cs

```diff
@@ -0,0 +1,84 @@
+﻿//----------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+#if ANDROID
+using System;
+using Android.App;
+#endif
```

Comment:
minor: generally I suggest putting using statements inside the namespace.

Reply:
That wouldn't be consistent w/the other classes with have currently...why do you suggest putting them inside?

Reply:
mostly to satisfy StyleCop rules... for most practical reasons it makes very little difference. Thus mostly a style reason (and most teams I worked on used that style). There is no real need to change these now - I just god used to seeing this the other way around... (and since the tools when enabled will force this behavior)

---
In reply to: [182148298](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1038#discussion_r182148298) [](ancestors = 182148298)

Reply:
interesting. I read some documentation on inside vs outside...seems like inside is slightly better, but I think it's important to have consistency at this point.